app/features/Kaichat/core.py

```diff
@@ -1,6 +1,6 @@
-from langchain_google_genai import GoogleGenerativeAI
+from langchain_google_vertexai import VertexAI

```

Comment:
Why is `langchain_google_genai.GoogleGenerativeAI` being changed to `langchain_google_vertexai.VertexAI` in another feature, while you're still using `langchain_google_genai.GoogleGenerativeAI` in your syllabus generator?